src/stories/components/Line/Line.story.tsx

```diff
@@ -141,7 +141,7 @@ const HistoricalCompareStory: StoryFn<typeof Line> = (args): ReactElement => {
 const LineWithVisiblePointsStory: StoryFn<typeof Line> = (args): ReactElement => {
 	const chartProps = useChartProps({ ...defaultChartProps, data: workspaceTrendsDataWithVisiblePoints });
 	return (
-		<Chart {...chartProps}>
+		<Chart {...chartProps} animations={false}>
```

Comment:
nit: let's be careful we don't let these slip into the storybook unless necessary